cardano-testnet/src/Testnet/Process/Cli/DRep.hs

```diff
@@ -28,20 +23,17 @@ import           Control.Monad (forM, void)
 import           Control.Monad.Catch (MonadCatch)
 import qualified Data.Aeson as Aeson
 import qualified Data.Aeson.Lens as AL
-import           Data.List (isInfixOf)
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import           Data.Word (Word32)
-import           GHC.IO.Exception (ExitCode (..))
 import           GHC.Stack
 import           Lens.Micro ((^?))
 import           System.FilePath ((</>))
 
 import           Testnet.Components.Query (EpochStateView, findLargestUtxoForPaymentKey,
                    getCurrentEpochNo, getMinDRepDeposit, waitUntilEpoch)
-import qualified Testnet.Process.Cli as H
+import           Testnet.Process.Cli.Transaction
 import qualified Testnet.Process.Run as H
```

Comment:
I've always been puzzled by those `as H` qualified imports that do not come from `hedgehog`, so this PR may be an occasion to fix them if you feel like it. This can also be done in a follow-up PR.

Reply:
done!